### ISSUE TITLE
"full-text search" donot work!

### ISSUE THREAD

**Issue by ShengChangJian** [christian-fei/Simple-Jekyll-Search#63]
When I use the search.json which not enable the full-text search, do work very well to search the post.
But When I replace the search.json to enable the full-text search, it donot work anymore and it cannot search nothing and even no feedback with "No find results" .
I donot know the reason.

**Comment by yo1995** [comment]
@ShengChangJian 应该是编码的问题，全英文就可以。正在看怎么改一下……
probably the coding problem. try to use utf-8

**Comment by yo1995** [comment]
```ruby
module Jekyll
  module CharFilter
    def remove_chars(input)
      input.gsub! '\\','&#92;'
      input.gsub! /\t/, '    '
	  input.gsub! '@',''
	  input.gsub! '$',''
	  input.gsub! '%',''
	  input.gsub! '&',''
	  input.gsub! '"',''
	  input.gsub! '{',''
	  input.gsub! '}',''
    end
  end
end

Liquid::Template.register_filter(Jekyll::CharFilter)

```

Tried to modify the filter module mannually

**Comment by yo1995** [comment]
```ruby
module Jekyll
  module CharFilter
    def remove_chars(input)
      input.gsub! '\\','&#92;'
      input.gsub! /\t/, '    '
	  input.gsub! '@',''
	  input.gsub! '$',''
	  input.gsub! '%',''
	  input.gsub! '&',''
	  input.gsub! '"',''
	  input.gsub! '{',''
	  input.gsub! '}',''
    end
  end
end

Liquid::Template.register_filter(Jekyll::CharFilter)
```

Manually modified version for json files with some restricts.
Hope the developer can ameliorate the existing plugin scripts..:confused:

**Comment by christian-fei** [comment]
@ShengChangJian thanks for your feedback!
Will look into this

**Comment by christian-fei** [comment]
Should be fixed with the new release [v1.0.5](https://github.com/christian-fei/Simple-Jekyll-Search/releases/tag/v1.1.5)
